specs/WinRT/WinRTAPIContracts.md

```diff
@@ -62,27 +74,47 @@ namespace Microsoft.Windows.System
     }
 }
 ```
-## 1.1 ship blocker
-Any APIs that changed between 1.0 and 1.1 need to add a contract or 1.1 will not ship.
 
-## Telemetry
-API specific telemetry will not be collected because it does not provide any additional information
-compared to the Windows App SDK version.
+# 4. Missing contracts is a ship blocker
+WinRT APIs without a contract are a shipblocking bug.
 
-## Gate check-ins
+## 4.1. Gate check-ins
 Because the contracts are used for API versions there will need to be work done in the build system
 to make sure
 1. All APIs have contracts
 2. Each contract is correct. This means version and name.
 3. All Contracts and dependencies work with each other.
-4. Contracts/Types in releases honor the source/binary/behavior compatibility rules e.g. 
+4. Contracts/Types in releases honor the source/binary/behavior compatibility rules e.g.
 contracts/types in a stable release doesn't change in future releases of the same major version
 (e.g. contracts/types in 1.1 don't change in 1.2, 1.3, ...)
 
 The builds that check for contracts compatibility are:
 1. Nightly
-2. Release 
+2. Release
 3. PR
 
-## Open issues
-None.
+# 5. Telemetry
+API contract version specific telemetry will not be collected because it does not provide any
+additional information compared to the Windows App SDK version.
+
+# 6. All Contracts in Windows App SDK
+The list of all contracts defined across Windows App SDK
+
+TODO:Start of the list. Needs review and undoubtedly correction and additions (e.g. IXP)
+
+| Feature            | Repository    | Contract                        | NamespaceMicrosoft.Windows.System.Power              | Comment  |
```

Comment:
NamespaceMicrosoft.Windows.System.Power?

Reply:
Are you looking at latest? I don't see that

![image](https://user-images.githubusercontent.com/72222/159044893-238c7063-c951-448f-852b-606226bdcc4f.png)